### ISSUE TITLE
Question regarding taxon-specific biases

### ISSUE THREAD

**Issue by jorondo1** [FrederickHuangLin/ANCOMBC#277]
Hi,

First of all, thanks for developing this awesome tool, I keep recommending it to colleagues.

We are testing differential abundance using a _sequence_ abundance table (as opposed to taxonomic abundances) as it was generated with a k-mer matching approach (sourmash, akin to kraken/bracken). As these tools generate abundances that are not normalized by genome length or ploidy, I was wondering if ANCOM-BC2's modelling of taxon-specific bias estimation would account for this. 

In other words, will the over-representation of species with longer genomes bias the fold-change estimations? I assume ANCOM's model corrects for this, but since the paper only talks about biases arising from the sequencing process, I'm wondering if there could be reasons why it would not apply or apply differently to biases introduced by the reads classification method. 

Thanks in advance!
Jo

**Comment by jorondo1** [comment]
Just a friendly ping, has anyone thought about this?
